bindings_ffi/tests/mvn2nix-lock.json

```diff
@@ -0,0 +1,1384 @@
+{
+  "dependencies": {
```

Comment:
this will cache all the kotlin/maven dependencies in CI. these never change so should be fast each time.